.github/workflows/close-stale-issues.yml

```
@@ -0,0 +1,25 @@
+name: Close Stale Issues
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  issues:
+    types: [opened, edited, labeled, unlabeled]
+
+jobs:
+  close-stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close stale issues
+        uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 30
+          days-before-close: 7
+          stale-label: stale
```

```suggestion
          stale-issue-label: Stale         # This setting defines the label that will be applied to issues that are considered stale
```

[`stale-label`](https://github.com/actions/stale) doesn't appear to be an option